packages/block-library/src/query/editor.scss

```diff
@@ -31,6 +31,7 @@
 		cursor: pointer;
 		margin: 0 $grid-unit-15 $grid-unit-15 0;
 		width: 200px;
+		max-height: 200px;
```

Comment:
I chose this value mostly at random — @jasmussen or @jameskoster, do you have a better idea for what this should be?

Reply:
Hmm, are we sure this isn't a bug somewhere in how these patterns are rendered? I don't see any reason for the thumbnails to be so tall. 🤔

Reply:
Yeah, after I opened this I saw that it only happens in Safari, so I wonder if there's a browser thing we're not accounting for. 🤔

Reply:
Yeah I'm also seeing weirdness here and assuming a max-height. I could swear this used to work without one. I wonder what regressed.